run_ci_tests.sh

```diff
@@ -2,55 +2,44 @@
 
 cd $(dirname `realpath $0`)
 
-# Start the container
-export PUBLIC_KEY='ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGzHvK2pKtSlZXP9tPYOOBb/xn0IiC9iLMS355AYUPC7'
-export DOCKER_MODS='linuxserver/mods:openssh-server-ssh-tunnel'
-
-name=openssh
-
-docker run \
-    --name $name \
-    --rm \
-    -d \
-    -p 127.0.0.1:2222:2222 \
-    -e 'USER_NAME=test-user' \
-    -e DOCKER_MODS \
-    -e PUBLIC_KEY \
-    linuxserver/openssh-server:amd64-latest
-
-function cleanup {
-    docker stop $name
-
-    # Revert modification to ~/.ssh/known_hosts
-    ssh-keygen -R "[127.0.0.1]:2222"
-
-    ssh-agent -k
-}
-trap cleanup EXIT
+./start_sshd.sh
 
 export RUSTFLAGS='--cfg=ci'
 
-cargo hack check --feature-powerset
+# Use a different target-dir since RUSTFLAGS='--cfg=ci' seems to
+# affect dependencies as well.
+#
+# Since IDEs usually do not set RUSTFLAGS='--cfg=ci', setting it
+# here would cause all the dependencies and openssh to be rebuilt.
+#
+# Thus it makes run_ci_tests.sh incrediably slow, and it also
+# affects IDEs checking, since now the IDEs also need to
+# rebuild the crate.
+CARGO_OPTS='--target-dir ci-target'
```

Comment:
```suggestion
export CARGO_TARGET_DIR=ci-target
```

That way you don't have to pass it around later.